packages/govuk-frontend/src/govuk/components/header/header.mjs

```diff
@@ -75,7 +76,7 @@ export class Header extends GOVUKFrontendComponent {
       throw new ElementError({
         componentName: 'Header',
         element: $menu,
-        identifier: `#${menuId}`
+        identifier: `List element #${menuId}`
```

Comment:
A more helpful descriptor might be something like "Nav menu" or "Menu element", but I've gone with this in the meantime since this is a `<ul>`

Reply:
From the **Header** guidance we have elements "Service name" and "Navigation"

Shall we use HTML to help here?

```suggestion
        identifier: `Navigation (\`<ul id="${menuId}">\`)`
```